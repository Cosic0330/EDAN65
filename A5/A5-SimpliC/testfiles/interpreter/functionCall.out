New put: Variable: return Value: 0
{return=0}
New put: Variable: returnValue Value: 0
{returnValue=0, return=0}
New put: Variable: 0_i Value: 10
{returnValue=0, 0_i=10, return=0}
New get: Variable: 0_i Value: 10
New get: Variable: return Value: 0
New get: Variable: 0_i Value: 10
New put: Variable: a Value: 10
{a=10}
New put: Variable: b Value: 20
{a=10, b=20}
New put: Variable: return Value: 0
{a=10, b=20, return=0}
New put: Variable: returnValue Value: 0
{a=10, b=20, returnValue=0, return=0}
New get: Variable: a Value: 10
New get: Variable: b Value: 20
Return value: 30
New put: Variable: return Value: 1
{a=10, b=20, returnValue=0, return=1}
New put: Variable: returnValue Value: 30
{a=10, b=20, returnValue=30, return=1}
New get: Variable: return Value: 1
New put: Variable: return Value: 0
{a=10, b=20, returnValue=30, return=0}
New get: Variable: returnValue Value: 30
30
returning
New get: Variable: return Value: 0
