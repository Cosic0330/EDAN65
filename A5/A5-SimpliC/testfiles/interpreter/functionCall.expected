New put: Variable: return Value: 0
{return=0}
New put: Variable: returnValue Value: 0
{returnValue=0, return=0}
New put: Variable: i Value: 10
{returnValue=0, i=10, return=0}
New get: Variable: i Value: 10
New get: Variable: return Value: 0
New get: Variable: i Value: 10
New put: Variable: a Value: 10
{a=10}
New put: Variable: b Value: 20
{a=10, b=20}
New put: Variable: return Value: 0
{a=10, b=20, return=0}
New put: Variable: returnValue Value: 0
{a=10, b=20, returnValue=0, return=0}
New get: Variable: a Value: 10
New get: Variable: b Value: 20
New put: Variable: return Value: 1
{a=10, b=20, returnValue=0, return=1}
New put: Variable: returnValue Value: 30
{a=10, b=20, returnValue=30, return=1}
New get: Variable: return Value: 1
New get: Variable: returnValue Value: 30
30
New get: Variable: return Value: 0
